### ISSUE TITLE
(opens in a new tab) causing issues for Dragon users

### ISSUE THREAD

**Issue by abbott567** [alphagov/govuk-design-system#1286]
The original guidance of `(opens in a new window)` was changed to `(opens in a new tab)` when it was put into the design system. 

We've found this hijacks a core function of Dragon. If you're using Dragon and you want to open a new tab, when you say `new tab` it's actually clicking any link on the page that says `(opens in a new tab)`. 

The problem is compounded on a service where the phase banner feedback link opens in a new tab, as for the entire duration of that service, the new tab command will click the feedback link.

**Comment by abbott567** [comment]
Probably should have mentioned it was on Dragon 14, Windows 10, IE11. 
Incase it's version specific.

**Comment by timpaul** [comment]
Thanks @abbott567 -  we'll investigate this more, create some reduced test cases and report the bug to Dragon, and then we'll decide how we might mitigate the effects of the bug for this component.

**Comment by selfthinker** [comment]
I have just tested this with Dragon Professional 15.3 in IE11 as well as the latest Chrome.

Saying "new tab" doesn't do anything.
Saying "open new tab" opens a new tab.
Saying "click new tab" clicks on a link that has the words "new tab" in it.

I have looked through the settings to find out if you can make it open a tab by just saying "new tab" but couldn't find anything.
The only thing I found was the option to:

> Require "Click" to select hyperlinks in HTML window

With that enabled saying "new tab" clicks on a link that has those words in it.

I assume that what @abbott567 describes either only happens in older versions of Dragon or there is a setting which I couldn't find that would have the same effect.

**Comment by selfthinker** [comment]
I should add, I have also tested saying "open [word]" to check if using the word "open" would click on a link. (For example, saying "open benefits" on the GOV.UK homepage to click on the "Benefits" link.) But that didn't do anything either.
I wonder if there is (or was) an option anywhere to redefine the word "click" to something else?

**Comment by 36degrees** [comment]
> Probably should have mentioned it was on Dragon 14, Windows 10, IE11.

@selfthinker do we have access to Dragon 14 to see if we can reproduce it in that version?

**Comment by selfthinker** [comment]
> @selfthinker do we have access to Dragon 14 to see if we can reproduce it in that version?

No, unfortunately you cannot have multiple versions of Dragon running at the same time.
Even if it was possible to install it in a virtual machine, I wouldn't know how to get my hands on an older version.

It's probably worth mentioning that Dragon 14 is not supported anymore.
So, reporting this as a bug to Nuance wouldn't achieve anything.

**Comment by selfthinker** [comment]
> It's probably worth mentioning that Dragon 14 is not supported anymore.

Actually, I was wrong about that. It *is* still supported.

**Comment by abbott567** [comment]
Oh no. Perhaps it was an issue specific to 14 that has now been fixed? 
I doubt I'll even be able to get a screen recording of it happening as it is running on a managed machine with a smart card etc.
So many variables =(

**Comment by 36degrees** [comment]
> The only thing I found was the option to:
> 
> > Require "Click" to select hyperlinks in HTML window
> 
> With that enabled saying "new tab" clicks on a link that has those words in it.

Is this the right way around? Or is this the case when that option is _disabled_?

**Comment by selfthinker** [comment]
> Is this the right way around? Or is this the case when that option is _disabled_?

Ah, no. You are right in your suspicion. It is the other way around. Saying "new tab" when that option was **disabled** would click on those links. It is enabled by default.

**Comment by 36degrees** [comment]
As we've not been able to reproduce this, we're going to close this issue.

@abbott567 if you are able to get a screen recording (even just by taking a video of the screen) that demonstrates this bug then we can re-open this and investigate further.

> We've found this hijacks a core function of Dragon. If you're using Dragon and you want to open a new tab, when you say `new tab` it's actually clicking any link on the page that says `(opens in a new tab)`.

We'd also need to see how the same command behaves correctly on other websites, as in @selfthinker's testing she was not able to get 'new tab' to open a new tab – only 'open new tab' worked.
